.github/workflows/e2e.yaml

```diff
@@ -33,9 +33,9 @@ jobs:
       run: |
         mkdir _output
 
-        docker build -t gcr.io/k8s-staging-kas-network-proxy/proxy-agent:master -f artifacts/images/agent-build.Dockerfile .
+        docker build -t gcr.io/k8s-staging-kas-network-proxy/proxy-agent:master -f artifacts/images/agent-build.Dockerfile --build-arg BASEIMAGE=golang:${{ env.GO_VERSION }} --build-arg RUNNERIMAGE=gcr.io/distroless/static-debian11 .
         docker save gcr.io/k8s-staging-kas-network-proxy/proxy-agent:master > _output/konnectivity-agent.tar
-        docker build -t gcr.io/k8s-staging-kas-network-proxy/proxy-server:master -f artifacts/images/server-build.Dockerfile .
+        docker build -t gcr.io/k8s-staging-kas-network-proxy/proxy-server:master -f artifacts/images/server-build.Dockerfile --build-arg BASEIMAGE=golang:${{ env.GO_VERSION }} --build-arg RUNNERIMAGE=gcr.io/distroless/static-debian11 .
```

Comment:
Since the released image is a rootless image, it will no longer be needed after the rootless e2e ci is repaired. Is my understanding correct?

Reply:
Yes. I could put a TODO here, saying to switch RUNNERIMAGE to match the Makefile default, once we get rootless e2e fixed.